### PR TITLE
add new storage queue for fhir test events and exceptions

### PR DESCRIPTION
**Pull request by BobanL**
# DEVOPS PULL REQUEST

## Related Issue

- Creates a queue for #4740 

## Changes Proposed

- Create new storage queue for fhir objects and exception handling

### CODE REVIEW

File: ops/dev/main.tf [hunk 1]
```diff
@@ -38,6 +38,16 @@ resource "azurerm_storage_queue" "test_event_exceptions_queue" {
...
+resource "azurerm_storage_queue" "flu_test_event_queue" {
+  name                 = "flu-test-event-publishing"
+  storage_account_name = azurerm_storage_account.app.name
+}
+
+resource "azurerm_storage_queue" "flu_test_event_exceptions_queue" {
+  name                 = "flu-test-event-publishing-exceptions"
+  storage_account_name = azurerm_storage_account.app.name
+}
+
```
Comment (BobanL) [inline ops/dev/main.tf:50]: @rin-skylight / @alismx I see the terraform plan looks OK to me. If I wanted to test this out and see the queue terraform creates, could I just deploy this to a dev environment and check in azure?

Comment (rin-skylight) [reply ops/dev/main.tf:50]: Yes, this is the recommended approach! If you can conduct a test, that would be greatly appreciated. I would like to make sure that things are successful before we attempt to run this against `prod`.

**Comment by BobanL** [comment]
@rin-skylight So this was deployed to dev4 and it created the queues successfully without effecting the existing queues, but since nothing uses these queues yet I can't quite test them. I did end up testing the standard queue and can confirm that this had no impact on that. 
image
image

**Review by rin-skylight — DISMISSED** [review]
The production plan for this looks to be sound, and tests are reported to be successful. Everything looks good! Thanks for putting this together! 🚀

**Comment by BobanL** [comment]
Renamed the storage queue to fhir since that's the approach we will be taking with the flu results.

**Review by rin-skylight — APPROVED** [review]

**Review by alismx — APPROVED** [review]
Glad you're using fhir in the name. This LGTM!

**Pull request merged**